acceptance_tests/features/steps/create_message_internal.py

```diff
@@ -2,25 +2,28 @@
 
 from acceptance_tests import browser
 from acceptance_tests.features.pages import create_message_internal
+from common.collection_exercise_utilities import find_case_by_enrolment_code
 
 
 @given("The internal user has found the associated respondent")
-def user_has_found_respondent(_):
-    assert create_message_internal.RESPONDENT_DETAILS.get_respondent_id() is not None
-    assert create_message_internal.RESPONDENT_DETAILS.get_ru_id() is not None
-    assert create_message_internal.RESPONDENT_DETAILS.get_ru_ref() is not None
+def user_has_found_respondent(context):
+    context.party_id = find_case_by_enrolment_code(context.iac)['partyId']
+
+    assert context.respondent_user_name is not None
+    assert context.party_id is not None
+    assert context.short_name is not None
 
 
 @when("they choose to send them a secure message and navigated to the 'send message' page")
 @given("the user is on the send message page")
-def navigate_to_send_message(_):
-    create_message_internal.go_to()
+def navigate_to_send_message(context):
+    create_message_internal.go_to(context)
 
 
 @then("the \'To\' field is populated with the respondent's name")
```

Comment:
See above regarding quotes in clause strings

Reply:
fixed